docs/articles/nunit/writing-tests/TestFixtureData.md

```diff
@@ -10,7 +10,7 @@ The following example varies the example shown under [TestFixture Attribute](xre
 a `TestFixtureSourceAttribute` with a data source in a separately defined class.
 
 ```csharp
-[TestFixtureSource(typeof(MyFixtureData), "Fixtureparams")]
+[TestFixtureSource(typeof(MyFixtureData), nameof(Fixtureparams))]
```

Comment:
```suggestion
[TestFixtureSource(typeof(MyFixtureData), nameof(MyFixtureData.Fixtureparams))]
```

I believe this needs to include the class name, otherwise we'll see an error about it not existing.

Reply:
Good catch, thank you!